mitodl/ocw-studio
# Sync videos from Google Drive files_final folder, don't transcode or upload to Youtube

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #1225 

#### What's this PR do?
- If videos are uploaded to the `files_final` Goole Drive folder for a site, they will now be synced instead of ignored.  However, they will be given a resource type of "Other" instead of "Video" and not transcoded or uploaded to YouTube.

#### How should this be manually tested?
- Add a video to the `files_final` folder, and another to the `videos_final` folder for a site
- Add an image or pdf to the `files_final` folder and another to the `videos_final` folder.
- Click the "Sync w/Google Drive" button
- The video in the `files_final` folder should be synced, have a resource type of "Other", and not have a corresponding `Video` object (check in django admin).
- The video in the `videos_final` folder should be synced, have a resource type of "Video", and have a corresponding `Video` object and `VideoJob` in Django Admin.
- The non-video file in the `videos_final` folder should still be ignored and not synced, the file in the `files_final` should be synced as usual.
